CONTRIBUTING.md

```diff
@@ -418,22 +420,23 @@ For example, the image referenced above actually lives in `superset-frontend/ima
 
 #### OS Dependencies
 
-Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.
+Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.  
+You also need to install MySQL or [MariaDB](https://mariadb.com/downloads).
 
-Ensure Python versions >3.7, Then proceed with:
+Ensure that you are using Python version ≥3.7, then proceed with:
```

Comment:
Do we want to restrict to Python 3.7 and 3.8 only?

Indeed Superset `1.1.0` isn't compatible Python 3.9 (at least due to [`psycopg2-binary==2.8.5`](https://github.com/apache/superset/blob/master/requirements/development.txt#L26) that [isn't itself Py3.9 compatible](https://github.com/psycopg/psycopg2/issues/1200#issuecomment-735237051)).

Reply:
Actually modified/fixed in c08d34e2c